app/styles/authors.scss

```diff
@@ -0,0 +1,61 @@
+.vert-align .form-group {
+    margin-top: 15px;
+}
+
+.bib-padding {
+    text-align: center;
+}
+
+// Author table divisions
+.author-table {
+    td {
+        border-top: 1px solid #ddd;
+    }
+}
+
+.col-division-left {
+    padding-left: 20px;
+    border-left: 1px solid #ccc;
+}
+
+.col-division-right {
+    border-right: 1px solid #ccc;
+}
+
+.search-box {
```

Comment:
In general, we encourage components to use more distinctive style names, to avoid interfering with other components. Eg https://github.com/abought/ember-osf/blob/1c3fa17675dcb2d5ad5b4fa6237da312d44242d2/addon/components/file-browser/style.scss#L1-L1